PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamZVtxMultContainer.cxx

```diff
@@ -160,6 +160,14 @@ void AliFemtoDreamZVtxMultContainer::PairParticlesSE(
             ResultsHist->FillMassQADist(HistCounter, RelativeK,
                                         itPart1->GetInvMass(),
                                         itPart2->GetInvMass());
+	    TVector3 momPart1 = itPart1->GetMomentum();
```

Comment:
Maybe that could go to the fill hist function and you pass only the Particles there
ResultsHist->FillPairInvMassQAD(HistCounter, itPart1, itPart2);